prometheus_speedtest/prometheus_speedtest.py

```diff
@@ -16,14 +16,14 @@
 
 flags.DEFINE_string('address', '0.0.0.0', 'address to listen on')
 flags.DEFINE_integer('port', 9516, 'port to listen on')
-flags.DEFINE_integer('server', 0, 'speedtest server to use - use 0 for auto-selection')
+flags.DEFINE_multi_integer('servers', [], 'speedtest server(s) to use - leave empty for auto-selection')
```

Comment:
You  want to leave this set to `None`, it's a different value than an empty list.

Python is weird.